documentation/ldap-guide.html

```diff
@@ -356,7 +356,7 @@ <h3>Manually Editing the Config File</h3>
 
     <pre>&lt;searchFilter&gt;&lt;![CDATA[(&(sAMAccountName={0})(|(givenName=GEORGE)(givenName=admin)))]]&gt;&lt;/searchFilter&gt;</pre>
 
-    <p><a name="ctxFactory"><h2>Custom Inital Context Factory</h2></a></p>
+    <h2 id="ctxFactory">>Custom Inital Context Factory</h2>
```

Comment:
stray '>' here?

Reply:
Good catch. Fixed it in the next PR (that addresses all files of the documentations directory).